oslabs-beta/vista
# Nestorcayananjr/refactor introspection query

# Issue Type


# Issue Description & Business Use Case

This PR consists of various refactoring including modularizing and partial typing of parseSchemaAndFormat, eliminating commented out code in various files, and reorganizing logic. The goal of this PR is for a cleaner codebase, to provide more descriptive and helpful error messages, and better code readability.

# Ticket Item

[Trello ticker](https://trello.com/c/gzHEQyJp)

# Steps to Reproduce Bug / Validate Feature / Confirm Tech Debt Fix

1. Pull latest changes
2. Run dev server
3. Confirm all features are working as normal

# Additional context

Strict typing in parseSchemaAndFormat needs to be completed. Current block is my own understanding of GraphQL's exported types and how to properly utilize them.
